README.md

```diff
@@ -23,7 +23,7 @@ options object.
   The maximum amount of time (seconds) the Reader will backoff for any single backoff
   event.
 * ```maxAttempts: 0``` <br/>
-  The number of times to a message can be requeued before it will be handed to the DISCARD handler and then automatically finished. 0 means that there is **no limit.** If no DISCARD handler is specified and `maxAttempts > 0`, then the message will be finished automatically when the number of attempts has been exhausted.
+  The number of times a given message will be attempted (given to MESSAGE handler) before it will be handed to the DISCARD handler and then automatically finished. 0 means that there is **no limit.** If no DISCARD handler is specified and `maxAttempts > 0`, then the message will be finished automatically when the number of attempts has been exhausted.
```

Comment:
The # of attempts is zero based so when `attempts >= maxAttempts`, it has already been attempted `maxAttempts` times. [See code here](https://github.com/dudleycarr/nsqjs/blob/26ea437f06ff4e881c905ff1b0e455fd6b2f57c7/src/reader.coffee#L150) and [test here](https://github.com/dudleycarr/nsqjs/blob/4a6a089ddfaa6d0d31370527b2904e3fbe4819cf/test/reader_test.coffee#L15).

You can see in the [nsq source code here](https://github.com/nsqio/nsq/blob/47b084805bd9a1e072e0547bda487408819b6ca9/nsqd/protocol_v2_test.go#L153) that the expected value of `message.attempts` is one _after_ the message has been handed to a consumer.